app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
```

Comment:
According to the checklist, when importing multiple classes/modules, it's better to use brackets `()` to enhance readability. Consider changing the import statement to follow this style. Here is an example:

python
from app.errors import (
    NotWearingMaskError, 
    VaccineError
)
```